### DISCUSSION TITLE
pytorch： multi workers

### DISCUSSION THREAD

**Discussion by nan2088** [mosecorg/mosec#87] [Q&A] [answered]
https://github.com/mosecorg/mosec/blob/1627a9bfae48c9f1ca564137fe202d17bbff5a69/examples/resnet50_server_pytorch.py#L49

Hi, for multi-instances usage, by default the pytorch use default cuda stream, making it not able to run instances concurrently. So how do you solve this problem? multi-process？

**Comment by lkevinzc** [comment, marked as answer]
Hi @ShiyangZhang , yes we use multi-process. All the concurrent requests will be queued and multiple processes will consume the queue
